tests/message-extensions/src/card.ts

```diff
@@ -161,6 +161,16 @@ export async function createDummyCards(searchQuery: string) {
       thumbnail: {
         title: item.title,
         text: item.description,
+        // When a user clicks on a list item in Teams:
+        // - If the thumbnail has a `tap` property: Teams will trigger the `message.ext.select-item` activity
+        // - If no `tap` property: Teams will insert the full adaptive card into the compose box
+        // tap: { 
+        //   type: "invoke",
+        //   title: item.title,
+        //   value: {
+        //     "option": index,
+        //   },
+        // },
       } as ThumbnailCard,
```

Comment:
use satisfies instead of as.

Reply:
I didn't know about that command - that's what I was intending for. Thanks